tools/tox_matrix.py

```diff
@@ -138,6 +140,8 @@ def get_matrix_item(env, global_libraries, global_string_parameters,
         sep = r"\\" if platform == "windows" else "/"
         item["pytest_flag"] += (
             rf"--cov-report=xml:${{GITHUB_WORKSPACE}}{sep}coverage.xml ")
+    if item["pytest"] == "true" and item["pytest-results-summary"] == "true":
+        item["pytest_flag"] += "--junitxml results.xml"
```

Comment:
Depending on how someone configures their `tox.ini`, I think it might be possible for this file to created in a temp directory, for example. So maybe it would be safer to configure it similar to the cov-report path (`${{GITHUB_WORKSPACE}}{sep}results.xml`)?